### PR TITLE
H264 profile fix

### PR DESCRIPTION
**Pull request by groupboard**
Fix janus_sdp_get_codec_pt_full so that it works when fmtp comes before rtpmap in the sdp, for vp9 and h264.

### CODE REVIEW

File: sdp-utils.c [hunk 1]
```diff
@@ -705,16 +705,22 @@ int janus_sdp_get_codec_pt_full(janus_sdp *sdp, const char *codec, const char *p
...
 					char profile_id[20];
 					g_snprintf(profile_id, sizeof(profile_id), "profile-id=%s", profile);
 					if(strstr(a->value, profile_id) != NULL) {
 						/* Found */
 						JANUS_LOG(LOG_VERB, "VP9 profile %s found --> %d\n", profile, pt);
-						return pt;
+						if(check_profile) {
+							return pt;
+						}
+						else {
```
Comment (lminiero) [inline sdp-utils.c:721]: Code style nit: please put the `else` on the same line as the closing bracket.

File: sdp-utils.c [hunk 2]
```diff
@@ -725,7 +731,12 @@ int janus_sdp_get_codec_pt_full(janus_sdp *sdp, const char *codec, const char *p
 					if(strstr(a->value, profile_level_id) != NULL) {
 						/* Found */
 						JANUS_LOG(LOG_VERB, "H.264 profile %s found --> %d\n", profile, pt);
-						return pt;
+						if(check_profile) {
+							return pt;
+						}
+						else {
```
Comment (lminiero) [inline sdp-utils.c:737]: Same here.

File: sdp-utils.c [hunk 3]
```diff
@@ -734,7 +745,12 @@ int janus_sdp_get_codec_pt_full(janus_sdp *sdp, const char *codec, const char *p
...
```
Comment (lminiero) [inline sdp-utils.c:751]: Same here.

**Comment by lminiero** [comment]
Just tested this and it looks good to me! As soon as those small code style nits are addressed, I'll merge :+1:

**Comment by groupboard** [comment]
Ok done.

**Comment by lminiero** [comment]
Thanks, merging :+1:

**Pull request merged**